pulp_rpm/app/modelresource.py

```diff
@@ -346,10 +353,145 @@ def set_up_queryset(self):
 
     class Meta:
         model = RpmRepository
-        import_id_fields = ('name',)
+        import_id_fields = ("name",)
         fields = ("name", "pulp_type", "description", "original_checksum_types", "sub_repo")
 
 
+class UpdateCollectionResource(QueryModelResource):
+    """
+    Resource for import/export of rpm_updatecollection entities.
+    """
+
+    update_record = fields.Field(
+        column_name="update_record",
+        attribute="update_record",
+        widget=ForeignKeyWidget(UpdateRecord, field="digest"),
+    )
+
+    def set_up_queryset(self):
```

Comment:
Is this a convention?  What calls it?

Reply:
QueryModelResource, which does some boilerplate for anything that wants to play the PIE game. See https://github.com/pulp/pulpcore/blob/master/pulpcore/plugin/importexport.py#L25